### DISCUSSION TITLE
How do i access internal database using prisma

### DISCUSSION THREAD

**Discussion by laohanme** [Dokploy/dokploy#799] [Q&A]
I setup my environment in 
![image](https://github.com/user-attachments/assets/4aa003ac-2b34-4791-b9fd-12096f0d3054)

During the build process always cuase this.
```
#12 15.59 Invalid `prisma.message.findMany()` invocation:
#12 15.59 
#12 15.59 
#12 15.59 Can't reach database server at `container-name:5432`
```

**Comment by Siumauricio** [comment]
Read here to see the workarounds: https://github.com/Dokploy/dokploy/issues/507
